graphql/schema.graphql

```diff
@@ -10,6 +10,7 @@ type Query {
   viewableProjectRefs: [GroupedProjects]!
   githubProjectConflicts(projectId: String!): GithubProjectConflicts!
   project(projectId: String!): Project!
+  permissions(userId: String!): Permissions!
```

Comment:
This should be a child of User 
https://github.com/evergreen-ci/evergreen/blob/00a4c8f3e8e4559cc23e04a019b6d1725c40c3e5/graphql/schema.graphql#L1447-L1452

It will have userId automatically passed in.